compiler/front/in_options.nim

```diff
@@ -213,6 +213,10 @@ type
       ## Historically and especially in version 1.0.0 of the language
       ## conversions to unsigned numbers were checked. In 1.0.4 they
       ## are not anymore.
+    optOldDoNode
```

Comment:
Do we need to provide backward compatibility support right now? As in - no reason to start piling up more dialects during refactoring.

Reply:
If you tell me to drop supporting the old behavior, I do it right now. I just did this backwards compatibility, because originally before I got banned, I wanted it to be a PR for Araq-Nim. And that is what I preserved here.